tests/logs/RegressionTests_jet.log

```diff
@@ -5323,3 +5323,84 @@ Checking test 159 atmaero_control_p8_rad_micro_intel results ....
   0: The maximum resident set size (KB)                   = 1726596
 
 Test 159 atmaero_control_p8_rad_micro_intel PASS
+Mon Jan 29 19:20:40 UTC 2024
```

Comment:
Why is there no report from the here from the initial run (where I'm guessing only the one test failed)? Is the initial test not completing it's report of the failure of the RT or did you edit out that part?

Reply:
I only appended the rerun using cat and >> into the log, it had been previously pushed like that. So I'd assume the failure of the test may have caused that incomplete final report, however I'm unsure on how the timeout/failure would've caused it to completely disappear. I'll keep an eye on the jet log in the next PR.

Reply:
OK. I'll be glad when we get @BrianCurtis-NOAA updates for pass/fail details merged.